performance-evaluation/google_sheets.py

```diff
@@ -4,113 +4,129 @@
 import readchar
 
 
-# Checks the last updated timestamp por template. In case it was modified, script code may have to be adapted
-def validate_updated_timestamp(spreadsheet, timestamp):
-	if spreadsheet.updated != timestamp:
-		print('Fecha de última modificación: ' + spreadsheet.updated)
-		print('Cancelando script. Archivo \'' + spreadsheet.title + '\' modificado, revisar script.')
-		print('')
-		exit()
+def create_sheet(google_credentials, title, folder_key):
+    print('Creando nuevo sheet...')
+    sheet = google_credentials.create(
+        title + ' - ', template=None, folder=folder_key)
```

Comment:
That closing paren belongs in a new line. Like

```suggestion
        title + ' - ', template=None, folder=folder_key
    )
```

Reply:
Done